.github/workflows/release-nuxt-ui-preset.yaml

```diff
@@ -0,0 +1,63 @@
+name: Release (preset-nuxt-ui)
+
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+    tags:
+      - 'nuxt-ui-preset-v*'
```

Comment:
@AmmarKaraja Similar to this we should change it in the default release.yaml to `tags: - 'v*'`